geowave-grpc-protobuf-shaded/src/main/java/mil/nga/giat/geowave/service/grpc/protobuf/util/ByteStringContainer.java

```diff
@@ -0,0 +1,23 @@
+package mil.nga.giat.geowave.service.grpc.protobuf.util;
```

Comment:
It is used by gRPC clients when making a call through the stub. Otherwise, we have another reliance on protobuf's ByteString (this is part of the shade/relocation strategy). In the future I believe we should actually look at using a shaded version of hbase Kent and I came across. It shades it's usage of protobuf 2.5 - that way anything moving forward can just include protobuf 3 no problem. At the moment we are kinda doing things backward (everything using protobuf 3 needs to shade at the moment).